src/test/ui/imports/extern-prelude-extern-crate-restricted-shadowing.stderr

```diff
@@ -13,15 +13,19 @@ error[E0659]: `Vec` is ambiguous (macro-expanded name vs less macro-expanded nam
 LL |         Vec::panic!();
    |         ^^^ ambiguous name
    |
-   = note: `Vec` could refer to a struct from prelude
-note: `Vec` could also refer to the crate imported here
+note: `Vec` could refer to the crate imported here
   --> $DIR/extern-prelude-extern-crate-restricted-shadowing.rs:5:9
    |
 LL |         extern crate std as Vec;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^
 ...
 LL | define_vec!();
    | -------------- in this macro invocation
+note: `Vec` could also refer to the struct defined here
+  --> $SRC_DIR/libstd/prelude/v1.rs:LL:COL
+   |
+LL | pub use crate::vec::Vec;
+   |         ^^^^^^^^^^^^^^^
```

Comment:
Interesting that we point at the reexport and not the original definition. That might be fine (and introduce the concept of the prelude).

Reply:
This happens for items from other crates that are available through reexports.
The distinction between real items and reexports is kinda lost when the decoding results reach resolve (due to being mostly unnecessary).